### DISCUSSION TITLE
Craft 4 Support?

### DISCUSSION THREAD

**Discussion by ademers** [solspace/craft-express-forms#84] [Questions & Answers] [answered]
What, if any, are your plans to upgrade Express Forms (Lite & Pro) to be compatible with Craft 4? Thanks!

**Comment by kjmartens** [comment, marked as answer]
Hi @ademers,

Craft 4 support is on the way. We currently have an alpha version that is being tested now and will likely go into beta early next week. 🙂

**Reply by ademers** [reply]
@kjmartens Good to know! Thanks for the quick reply.

**Comment by kjmartens** [comment]
Express Forms 2.0 beta is now available 🙂

**Reply by bjprodneyl** [reply]
Very happy that Express Forms 2 Beta is available. Are there any plans to support recaptcha?

**Reply by kjmartens** [reply]
Hi @bjprodneyl,

We do plan to revisit Express Forms in the coming months and likely add more features to it, including more reCAPTCHA options. 🙂
